Userland/DevTools/UserspaceEmulator/ValueWithShadow.h

```diff
@@ -24,11 +24,16 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#pragma once
+
 #include <AK/Format.h>
 #include <AK/Platform.h>
+#include <LibX86/Types.h>
+#include <string.h>
 
-#pragma once
-
+#define INITIALIZED_64 0x01010101'01010101LLU
+#define INITIALIZED_128 u128(INITIALIZED_64, INITIALIZED_64)
+#define INITIALIZED_256 u256(INITIALIZED_128, INITIALIZED_128)
```

Comment:
Can we make these `constexpr` values instead?

Reply:
the byte access in in u256 would be a cast of `this` to the corresponding type, but yes

this will be fixed in the next commit iteration